SCRAPERS-LIST.md

```diff
@@ -1417,7 +1417,7 @@ watchreal.com|Hustler.yml|:heavy_check_mark:|:x:|:x:|:x:|CDP|-
 watchyoujerk.com|WatchYouJerk.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
 waybig.com|Waybig.yml|:heavy_check_mark:|:x:|:x:|:heavy_check_mark:|-|-
 wcpclub.com|AdultEmpireCash.yml|:heavy_check_mark:|:x:|:heavy_check_mark:|:x:|-|-
-wearehairy.com|wearehairy.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
+wearehairy.com|wearehairy.yml|:heavy_check_mark:|:heavy_check_mark:|:x:|:x:|-|-
```

Comment:
Needs a checkmark for the fourth column as well since you've added Performers to this scraper

You can also change the entry in the second column to indicate that you've changed the casing of the file to `WeAreHairy.yml`